src/controller/CHIPDeviceController.cpp

```diff
@@ -2071,6 +2074,18 @@ void DeviceCommissioner::SendCommissioningCompleteCallbacks(NodeId nodeId, const
     }
 }
 
+void DeviceCommissioner::ContinueCommissioningOverOperationalNetwork()
+{
+    ChipLogProgress(Controller, "DeviceCommissioner::ContinueCommissioningOverOperationalNetwork");
+
+    // This method is meaningful only for the state kWaitForDeviceInstallation
+    VerifyOrReturn(mCommissioningStage == kWaitForDeviceInstallation);
```

Comment:
The issue with logic is that it is assuming the commissioner will remember this and be persistent for until the device might be powered up. This could be a very long time and the commissioner most definitely will not be in this state. We need a design that lets you complete phase1 and resume the last stages at some point in the future.

Reply:
I will come up with a counter proposal soon.